tests/forte/data/data_store_serialization_test.py

```diff
@@ -0,0 +1,585 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for data store related operations.
+"""
+
+import logging
+import unittest
+import timeit
+import tempfile
+import os
+from sortedcontainers import SortedList
+from forte.data.data_store import DataStore
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class DataStoreTest(unittest.TestCase):
```

Comment:
We need test cases with different boolean combinations, and did we have the test cases where the fields are incorrect, missing, or wrong orders? Need to make these tests a bit clear to reflect what's being tested (by re-organizing, rename the functions, write comments).

Reply:
fixed by adding comments in test cases.